EXAMPLES.md

```diff
@@ -48,12 +58,60 @@ Output example :
 ```json
 {
     "account": {
-        "arn": "arn:aws:iam::685038099695:/john/",
-        "canonicalId": "8LTVNPCFXUUU8SRQCX3PJQ5PDMYP6TE7610KELF480Q4OWH9OZCS",
-        "id": "685038099695",
+        "arn": "arn:aws:iam::235437388852:/john/",
+        "canonicalId": "8dfc0d36337c3af81417d13e9a9fc20ec4efb553bdd70e27110d13632f74dad0",
+        "id": "235437388852",
         "emailAddress": "john@acme.com",
         "name": "john",
-        "createDate": "2016-07-02T21:26:38Z"
+        "createDate": "2019-12-26T10:49:20Z",
+        "quotaMax": 0
+    }
+}
+```
+
+### Create account with quota
+
+```sh
+vautclient create-account --name john2 --email john2@acme.com --quota 100
+```
+
+The previous example will create an account named 'john2' with an email
```

Comment:
examples don't do, they show. I would change this to: 
The following example shows creating an account named "john2", with an email address of "john2@acme.com" and a quota set to 100 bytes.